CriteoPublisherSdk/Sources/CR_CdbBid.h

```diff
@@ -74,6 +79,8 @@
 - (BOOL)isExpired;
 - (BOOL)isValid;
 
+- (void)setDefaultTtl;
```

Comment:
I suggest adding the time unit to the method name.

Reply:
I'm not sure this is something we want to do. This method basically sets the Default TTL into in inner state, from outside caller this is regardless of the underlying unit which is an implementation detail.